### DISCUSSION TITLE
確認済みのメッセージについて

### DISCUSSION THREAD

**Discussion by esperanzality** [Chipppppppppp/LIME#208] [Q&A（質問）]
こちらの確認済みのメッセージとメッセージの削除はどのうような機能でしょうか？
説明欄に記載ありましたら、申し訳ございません。
![Screenshot_20241113_143622_LINE](https://github.com/user-attachments/assets/7548222d-f5c8-4c96-8099-9f21e3b07604)
](url)

**Comment by areteruhiro** [comment]
取り消されたメッセージを保存機能で、
表示後
OKボタンを押すと「確認済みのメッセージ」に移動されます。

**Comment by esperanzality** [comment]
よく理解できました！ありがとうございます😊
